examples/speed_tester/speed_tester.c

```diff
@@ -470,12 +477,16 @@ nf_setup(struct onvm_nf_info *nf_info) {
                                 uint64_t *ts = (uint64_t *)rte_pktmbuf_append(pkt, sizeof(uint64_t));
                                 *ts = 0;
                         }
+
+                        /* New packet generated successfully */
                         pkts[i] = pkt;
+                        pkts_generated++;
                 }
-                onvm_nflib_return_pkt_bulk(nf_info, pkts, packet_number);
+                onvm_nflib_return_pkt_bulk(nf_info, pkts, pkts_generated);
 #ifdef LIBPCAP
         }
 #endif
+        packet_number = pkts_generated;
```

Comment:
if packet_number==0
   //no packets, print error msg, cry, exit

Reply:
Actually, check if we wanted to generate 0 packets. If we did than we're fine, but If we wanted 128 packets and got 0 we should just error out and exit